packages/yourloops/components/dashboard-widgets/medical-files/medical-files-widget.tsx

```diff
@@ -0,0 +1,96 @@
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { makeStyles } from "@material-ui/core/styles";
+
+import AssignmentIcon from "@material-ui/icons/Assignment";
+import Card from "@material-ui/core/Card";
+import CardHeader from "@material-ui/core/CardHeader";
+import CardContent from "@material-ui/core/CardContent";
+import MedicalRecordList from "./medical-record-list";
+import PrescriptionList from "./prescription-list";
+import { Patient } from "../../../lib/data/patient";
+import { useTeam } from "../../../lib/team";
+import WeeklyReportList from "./weekly-report-list";
+
+const useStyle = makeStyles(() => ({
+  cardContent: {
+    maxHeight: 450,
+    overflow: "auto",
+  },
+  medicalFilesWidget: {
+    width: "400px",
+    height: "fit-content",
+  },
+  medicalFilesWidgetHeader: {
+    textTransform: "uppercase",
+    backgroundColor: "var(--card-header-background-color)",
+  },
+}));
+
+export interface MedicalFilesWidgetProps {
+  patient: Patient;
+  userRole: string;
+}
+
+export interface CategoryProps {
+  teamId: string;
+  patientId: string;
+}
+
+export default function MedicalFilesWidget(props: MedicalFilesWidgetProps): JSX.Element {
+  const { t } = useTranslation("yourloops");
+  const classes = useStyle();
+  const { getRemoteMonitoringTeams } = useTeam();
+  const { patient } = props;
+
+  const team = getRemoteMonitoringTeams().find(team => team.members.find(member => member.user.userid === patient.userid));
+
+  if (!team) {
+    throw Error(`Could not find monitoring team for patient with id: ${patient.userid}`);
+  }
+
+  return (
+    <React.Fragment>
```

Comment:
Can you explain why it is useful to have a <React.Fragment>?

Reply:
just forgot to remove it !